src/cli.ts

```diff
@@ -245,21 +246,42 @@ program
     `List all locations to run the synthetics monitors. Pass optional '--url' and '--auth' to list private locations.`
   )
   .option('--url <url>', 'Kibana URL to fetch all public and private locations')
-  .option(
-    '--auth <auth>',
-    'API key used for Kibana authentication(https://www.elastic.co/guide/en/kibana/master/api-keys.html).'
+  .addOption(
```

Comment:
can we reuse the auth option from the push command and add mandatory only on push.